modules/kubernetes/eks-core/modules.tf

```diff
@@ -409,3 +409,15 @@ module "vpa" {
 
   source = "../../kubernetes/vpa"
 }
+
+module "node_local_dns" {
+  depends_on = [module.opa_gatekeeper, module.prometheus]
+
+  for_each = {
+    for s in ["node-local-dns"] :
+    s => s
+    if var.node_local_dns_enabled
+  }
+
```

Comment:
You need to overwrite the default ip, they differ between EKS and AKS in our current default setup.

```shell
$ kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}
172.20.0.10
```

Please add:

dns_ip = "172.20.0.10"